internal/pkg/cli/cmd/cmd.go

```diff
@@ -208,6 +208,7 @@ func NewRootCommand(stdin io.Reader, stdout io.Writer, stderr io.Writer, prompt
 		root.addAlias(`t`, `template`)
 		root.addAlias(`r`, `template repository`)
 		root.addAlias(`repo`, `template repository`)
+		root.addAlias(`template test`, `template test run`)
```

Comment:
Myslim si, ze toto nepotrebujeme, nizsie to mas spravne vyriesene `return RunCommand(d).RunE(cmd, args)`.